scripts/setup-ubuntu.sh

```diff
@@ -114,9 +114,19 @@ function install_fbthrift {
 
 function install_conda {
   mkdir -p conda && cd conda
-  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  ARCH=$(uname -m)
+  
+  if [ "$ARCH" == "x86_64" ]; then
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
```

Comment:
Can we coalesce the `if` and `if-else` by using
 `wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(ARCH).sh`

Reply:
Merged the wget for x86_64 and aarch64.